From: Don Dutile <ddutile@redhat.com>
Date: Mon, 28 Jul 2008 14:18:28 -0400
Subject: [xen] ia64 PV: Kconfig additions
Message-id: 488E0D74.3000005@redhat.com
O-Subject: Re: [RHEL5.3 PATCH 3/4]: Kconfig addition to build XEN PV-on-HVM drivers in bare metal / FV guest kernel
Bugzilla: 442991
RH-Acked-by: Bill Burns <bburns@redhat.com>
RH-Acked-by: Markus Armbruster <armbru@redhat.com>

Don Dutile wrote:
> BZ 442991 -- Include xenpv-driver in bare metal kernel rpm.
>
> Kconfig addition to build XEN PV-on-HVM drivers in bare metal / FV guest kernel.
> Mimics what was done for i386 & x86_64.
>
> See 0/4 for test details.
>
> Please review and ACK.
>
> - Don
>
>
>
>

diff --git a/arch/ia64/Kconfig b/arch/ia64/Kconfig
index 278056e..4e33a1d 100644
--- a/arch/ia64/Kconfig
+++ b/arch/ia64/Kconfig
@@ -639,3 +639,8 @@ config XEN_DEVMEM
 endif
 
 source "drivers/xen/Kconfig"
+
+if !XEN
+source "drivers/xenpv_hvm/Kconfig"
+endif
+
